content/en/seachat/getting-started/02-quick-start.md

```diff
@@ -0,0 +1,114 @@
+
+Feel free to try out the different ways to upload data to your knowledge base. For the purposes of this tutorial, we'll focus on utilizing the spreadsheet upload method. However, if you're interested in exploring additional data uploading techniques, we invite you to consult the [Add Knowledge]( /en/seachat/seachat-intro/03-add-knowledge/add-knowledge-intro) section within our user manual for comprehensive instructions and tips.
+<br/>
+<img width="80%" style="border-radius: 0.4rem" src="/images/product-updates/seachat/en/tutorial-intro/20240311-tutorial-intro-step3.png" alt="">
+<br/>
+
+## Step  4:  Upload Files to Knowledge Base
+Uploading a spreadsheet is as simple as clicking on the upload icon and choosing the desired spreadsheet. Once you see your uploaded files in the preview section below the drag-and-drop section, click on the **Next** button to confirm. 
+
+<br/>
+<img width="80%" style="border-radius: 0.4rem" src="/images/product-updates/seachat/en/tutorial-intro/20240311-tutorial-intro-step4.png" alt="">
+<br/>
+
+<br/>
+
+Navigate to the **Existing** section in the top-right corner of the dashboard once your files have been successfully uploaded.
+
+<br/>
+<img width="80%" style="border-radius: 0.4rem" src="/images/product-updates/seachat/en/tutorial-intro/20240311-tutorial-intro-step4-2.png" alt="">
+<br/>
+
+<br/>
+
+That’s it. Now, your SeaChat agent has processed the uploaded data, and let’s set it up for a test to see if the AI agent needs further fine-tuning. Click on the **Test AI Agent** button located in the bottom-right corner. Test away and let’s fine-tune the AI agent for optimal performance!.
+
+<br/>
+<img width="80%" style="border-radius: 0.4rem" src="/images/product-updates/seachat/en/tutorial-intro/20240311-tutorial-intro-step4-3.png" alt="">
+<br/>
+
+## Step  5:  Upload Files to Knowledge Base
+
+Before deploying the AI agent we've just created, it’s crucial to train it through testing and fine-tuning. Imagine yourself as the user interacting with the chatbot. You can now start asking any questions to see how your SeaChat agent responds.
+
+<br/>
+<img width="80%" style="border-radius: 0.4rem" src="/images/product-updates/seachat/en/tutorial-intro/20240311-tutorial-intro-step5.png" alt="">
+<br/>
+
+<br/>
+
+Your role as a tester involves verifying if our agent produces the correct responses based on the data uploaded to its knowledge base, and determining if the responses require linkage with external data or additional information. To review a message, hover over it and provide positive feedback to your agent with a thumbs up. If the message needs further review, give it a thumbs down.
+
+<br/>
+<img width="80%" style="border-radius: 0.4rem" src="/images/product-updates/seachat/en/tutorial-intro/20240311-tutorial-intro-step5-2.png" alt="">
```

Comment:
when i look at the image here, I can't see the details at all... How do you fix this?

![image](https://github.com/seasalt-ai/wiki/assets/543547/9ba07dfa-3376-4a74-9629-2fa1baabdec1)

Reply:
zoom in to the conversation so you can show the highlighted conversation. Like this
<img width="552" alt="Screenshot 2024-03-11 at 17 44 59" src="https://github.com/seasalt-ai/wiki/assets/20276372/63d6767f-3c09-4d1d-b967-606b3613d1e4">